### ISSUE TITLE
Volume level resets at random

### ISSUE THREAD

**Issue by LRitzdorf** [ytmdesktop/ytmdesktop#945]
### Consent

- [X] I verified that there is no open issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

While playing music, the volume level resets to some default level. This can occur at random, but also reliably happens when a new song begins. The volume change is not reflected by the volume slider in the YouTube Music interface, or by the output of `playerctl volume`.

Also, using either volume keyboard shortcut (that is, up or down) changes the volume with respect to the level shown on the slider — that is, if the volume was originally low (and is brought higher by the reset), using the volume-up shortcut actually lowers the output volume again. Similarly, if the volume is initially high (and is brought lower by the reset), using the volume-down shortcut will increase the volume.

### Expected Behavior

Volume level should not change randomly.

### Steps To Reproduce

1. Play a song.
2. Set the volume low (optional; makes the reset more noticeable).
3. Wait for a volume reset to occur at random, or when a new song begins.
4. Use the keyboard shortcut to adjust the volume level upward. The output volume will drop to the proper level, based on its value before the reset.

### YTMDesktop

1.14.1-1

### OS

Arch Linux

### OS Version

Rolling (currently on kernel 5.17.3-arch1-1)

### Arch

x64

### Installation way

AUR

### Anything else?

This issue seems to have appeared around the same time that #940 was resolved. I would test whether it still occurs with the new dB volume mode disabled, but due to #916, I can't access the Settings window.

**Comment by Alipoodle** [comment]
Firstly, we've not released a 1.14.1-1 ? (I presume this is some Arch Patch or something instead)

I've noticed this issue before in older versions I.E. 1.13 & 1.14 this was a problem, but this was fixed by forcing an update event on the slider which caused the video to also update it's volume correctly as well.
I've personally been running 1.14.1 for a while now and have not noticed anything going wrong with volume changing (especially while at low volumes) resetting randomly between or during a song.

**Comment by LRitzdorf** [comment]
Interesting. `1.14.1-1` is the version of the `ytmdesktop` AUR package — I can't check the setting panel, again due to #916. I tried `ytmdesktop --version` at the command line, but no dice. If there's another way to check the version I'm running, I'd be glad to try it.

Also, it's possible that strange things are going on with the AUR package in general, since I'm also seeing issues like this:
![image](https://user-images.githubusercontent.com/42657792/161872853-afc0710c-d5f4-4931-b4e7-b4d2819ed055.png)

I'll try to build manually, as well as testing the `ytmdesktop-git` package, in case either of those help.

**Comment by LRitzdorf** [comment]
When I attempt to build manually, the process hangs at the `rebuilding native dependencies  dependencies=@nodert-win10-rs4/windows.foundation@0.4.4 [more dependencies...]` stage, when running `npm install`. However, the AUR package still builds and installs cleanly.

When testing the `ytmdesktop-git` package (versus the standard `ytmdesktop`), the volume issue does go away, though other problems appear (notably, ads and their "skip" buttons don't render). Thus, for now, the bleeding-edge version isn't a viable option for me. It seems I might just have to wait until a new version is released, incorporating the volume fix without the ad issue.

**Comment by Alipoodle** [comment]
#916 wouldn't really help as it uses the `package.json` which hasn't been updated (I plan to fix this next Nightly)
`ytmdesktop --version` hasn't been setup. 😅 (probably should do)

Sounds like something is wrong with your `package.json` as you're trying to install a Window package.

Unaware of any Issues with the Ad button not showing... but I have Premium currently so I'm unable to see ads.  
I've not seen any reports for this either. 😅 Nor have we touched anything to this in the Live build yet.

**Comment by Alipoodle** [comment]
OH I forgot about the String in the Context Menu.
That is an issue with Development not downloading the Locale strings. This appears that it has failed to load it from GitHub, and without more information I don't really know what to do for helping on this.

**Comment by LRitzdorf** [comment]
Well, since the volume issue didn't appear in the bleeding-edge (`ytmdesktop-git`) build, we might just have to wait for the next release. I can manage that.

For the context menu strings, is there any log output (from the build process, presumably) I can provide that might help?

**Comment by sergiotca** [comment]
> OH I forgot about the String in the Context Menu. That is an issue with Development not downloading the Locale strings. This appears that it has failed to load it from GitHub, and without more information I don't really know what to do for helping on this.

Hi Alipoodle, I'm also experiencing the menu language issue.

This shows whenever I launch ytmdesktop from a terminal (among other stuff):

```
[!] To add translation for your introduced feature, you may navigate to 
[!] /home/sergiotca/.config/youtube-music-desktop-app/locales to modify the your locale file.
[!] Then kindly open a PR to ytmdesktop-locales repo. :)
```

This is the content of my folder ~/.config/youtube-music-desktop-app/locales:                        
en.json pt.json

My local language is Spanish. Shouldn't I see an "es.json" file or similar too?

I've also hasn't been available to access the setting menu since update 1.13. It gets stuck loading.
The same happens to the mini player and the lyrics window.

Using Manajaro and YTMD 1.14.1-1 (AUR "ytmdesktop-git").

**Comment by LRitzdorf** [comment]
All right, the volume issue is fixed for me in v1.14.2. The context menu strings are still weird, but that should probably be dealt with in a separate issue.
